<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuAdd_Drillset">
     <property name="title">
      <string>Add Drillset</string>
     </property>
     <addaction name="actionFrom_Image"/>
     <addaction name="actionEmpty_Set"/>
     <addaction name="actionCopy_Current"/>
    </widget>
    <widget class="QMenu" name="menuNew">
     <property name="title">
      <string>New</string>
     </property>
     <addaction name="actionBand"/>
     <addaction name="actionShow"/>
    </widget>
    <addaction name="menuNew"/>
    <addaction name="actionOpen"/>
    <addaction name="menuAdd_Drillset"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionInstrumentation"/>
    <addaction name="actionField"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionPerformer"/>
    <addaction name="actionDirector"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionFrom_Image">
   <property name="text">
    <string>From Image...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionEmpty_Set">
   <property name="text">
    <string>Empty Set</string>
   </property>
  </action>
  <action name="actionCopy_Current">
   <property name="text">
    <string>Copy Current</string>
   </property>
  </action>
  <action name="actionInstrumentation">
   <property name="text">
    <string>Instrumentation</string>
   </property>
  </action>
  <action name="actionField">
   <property name="text">
    <string>Field</string>
   </property>
  </action>
  <action name="actionBand">
   <property name="text">
    <string>Band</string>
   </property>
  </action>
  <action name="actionShow">
   <property name="text">
    <string>Show</string>
   </property>
  </action>
  <action name="actionPerformer">
   <property name="text">
    <string>Performer</string>
   </property>
  </action>
  <action name="actionDirector">
   <property name="text">
    <string>Director/Audience</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
